src/libutil/serialise.cc

```diff
@@ -194,10 +194,6 @@ size_t StringSource::read(char * data, size_t len)
 }
 
 
-#if BOOST_VERSION >= 106300 && BOOST_VERSION < 106600
-#error Coroutines are broken in this version of Boost!
-#endif
```

Comment:
So please turn that into a `src/libutil/meson.build` somehow per the above, otherwise this looks good to me.